packages/embed/src/factories/create-popup/create-popup.ts

```diff
@@ -97,7 +98,7 @@ export const createPopup = (formId: string, userOptions: PopupOptions = {}): Pop
       popup.style.opacity = '0'
       wrapper.style.opacity = '0'
       setTimeout(() => {
-        popup.parentNode.removeChild(popup)
+        popup.parentNode && popup.parentNode.removeChild(popup)
```

Comment:
Could be replaced with a safe navigation operator here:

```suggestion
        popup.parentNode?.removeChild(popup)
```

Reply:
theres the `unmountElement` function too